pythoncz_tests/models/github_test.py

```diff
@@ -0,0 +1,432 @@
+import re
+import warnings
+
+import pytest
+import requests
+import responses
+from werkzeug.contrib.cache import NullCache
+
+from pythoncz.models import github as github_module
+from pythoncz_tests.models import github_test_fixtures as fixtures
+
+
+@pytest.fixture()
+def github():
+    """Provides the 'github' module with caching disabled"""
+    original_cache = github_module.cache
+    github_module.cache = NullCache()
+    yield github_module
```

Comment:
Yup, this is special pytest behavior. You write tear down code after `yield` and that's it. Not sure about E tests, would need to see docs.